dts/bindings/rng/nordic,nrf54lcracen-ctrdrbg.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2025, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Nordic nRF54L CRACEN CTR_DRBG based (Random Number Generator)
+
+    This driver is only compatible with 54L devices, and may only be used from one processor
+    core. This driver cannot be used in conjunction with the nRF security PSA solution, as both
+    would attempt to use the CRACEN HW exclusively; When that is enabled, zephyr,psa-crypto-rng
+    should be selected instead.
```

Comment:
Not sure if driver constraints should be included in binding description.
Perhaps it would be better to note that this node is used to describe a subset of CRACEN resources, ie. CRACEN TRNG and CryptoMaster.

Reply:
> Perhaps it would be better to note that this node is used to describe a subset of CRACEN resources, ie. CRACEN TRNG and CryptoMaster.

This binding is used to describe the CTR_DRBG itself. Not the TRNG or Cryptomaster.